### DISCUSSION TITLE
How do I explicitly save the server?

### DISCUSSION THREAD

**Discussion by strikef** [thijsvanloef/palworld-server-docker#44] [Q&A] [answered]
Do we have a specific command for saving, or does it get triggered by action of some sort?

Also, how do I 'gracefully' stop the server? Like, save and quit safely...

**Comment by thijsvanloef** [comment, marked as answer]
This can be done using RCON, however I am still building that functionality. This pull request should solve your issue: https://github.com/thijsvanloef/palworld-server-docker/pull/42

**Reply by strikef** [reply]
Cool! Hope this patch lands quickly :o Thank you!

**Reply by thijsvanloef** [reply]
@strikef RCON has been added, you can now use commands to shut down the server: 
https://github.com/thijsvanloef/palworld-server-docker?tab=readme-ov-file#using-rcon

**Reply by strikef** [reply]
Great to hear! That was quicker than I expected.
Thanks a lot for this project. I and my friends are really enjoying the game using it :)

**Reply by thijsvanloef** [reply]
Glad to hear you are enjoying it :D
